canonical-web-and-design/canonical.com
# Add back /secure-boot-master-ca.crl

## Done

At some point we lost the CRL file on canonical.com (see #302)

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8002/secure-boot-master-ca.crl
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- See that the binary file downloads


## Issue / Card

Fixes #  231
